.changelog/2872.txt

```diff
@@ -0,0 +1,28 @@
+```release-note:note
```

Comment:
can you add a line in TPF readme file so we don't forget?

Reply:
done